### ISSUE TITLE
Checking out a change with a locally stored older version fails and falls back on the old version

### ISSUE THREAD

**Issue by ar-turek** [uwolfer/gerrit-intellij-plugin#235]
When I create a change in Gerrit and then use the plugin to checkout, a new branch gets created (review/artur_ciesielski/2806). This is fine.

But then pushing a new patch set and checking out again gives me the message (after fetching the change to FETCH_HEAD):
`git -c core.quotepath=false checkout -b review/artur_ciesielski/2806 FETCH_HEAD --`
`fatal: A branch named 'review/artur_ciesielski/2806' already exists.`
After which the plugin checks out the old (locally stored) branch (checked via Version Control console):
`git -c core.quotepath=false checkout review/artur_ciesielski/2806 --`

So basically I can't checkout a change without removing the local branch first.

PhpStorm version 2016.2 (#PS-162.1121.38)
Gerrit Code Review version 2.12.3
Gerrit Plugin version 0.9.8.2-146

**Comment by ar-turek** [comment]
@uwolfer Hey, can we expect this issue to be resolved in the near future? I could try to cobble together a pull request for this but, being no real Java programmer I am kind of reluctant to experiment with this.

**Comment by uwolfer** [comment]
@artur-ciesielski: I have worked on a solution, but it is not ready yet. I hope to get it done soon (but it will take some weeks).

**Comment by ar-turek** [comment]
@uwolfer Thank you for the update, that is all I really hoped for.

**Comment by eekboom** [comment]
I think this is the same problem as #267.
What do you think of the simple solution of just always including change number and patch set number in the branch name?
